eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -50,73 +49,92 @@ final class OptCachedTest {
+     *  different tests use the same code
+     *  { @code Files.setLastModifiedTime(
+     *  res,
+     *  FileTime.fromMillis(System.currentTimeMillis() + time)
+     *  ); }.
+     *  We need to think about how to remove this duplication.
      */
-    @Disabled
     @Test
-    void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path tmp) throws IOException {
-        final XML program = OptCachedTest.program(ZonedDateTime.now());
-        OptCachedTest.save(tmp, program);
+    void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path cache, @TempDir final Path dir)
+        throws Exception {
+        final XML xml = OptCachedTest.program();
+        final Path program = OptCachedTest.save(dir, xml);
+        OptCachedTest.save(cache, xml);
         MatcherAssert.assertThat(
             "We expected that the program will be returned from the cache.",
             new OptCached(
                 path -> {
                     throw new IllegalStateException("This code shouldn't be executed");
                 },
-                tmp
-            ).apply(program),
-            Matchers.equalTo(program)
+                cache,
+                program
+            ).apply(xml),
+            Matchers.equalTo(xml)
         );
     }
 
-    @Disabled
     @Test
-    void returnsFromCacheButTimesSaveAndExecuteDifferent(@TempDir final Path tmp)
-        throws IOException {
-        final XML program = OptCachedTest.program(ZonedDateTime.now().minusMinutes(2));
-        OptCachedTest.save(tmp, program);
+    void returnsFromCacheButLastModifiedTimesDifferent(
+        @TempDir final Path cache,
+        @TempDir final Path dir
+    )
+        throws Exception {
+        final XML xml = OptCachedTest.program();
+        final Path program = OptCachedTest.save(dir, xml);
+        OptCachedTest.save(cache, xml, 2000);
         MatcherAssert.assertThat(
-            "We expected that the not immediately saved program will be returned from the cache.",
+            "We expected that the program will be returned from the cache.",
             new OptCached(
                 path -> {
                     throw new IllegalStateException("This code shouldn't be executed");
                 },
-                tmp
-            ).apply(program),
-            Matchers.equalTo(program)
+                cache,
+                program
+            ).apply(xml),
+            Matchers.equalTo(xml)
         );
     }
 
-    @Disabled
     @Test
-    void returnsFromCacheCorrectProgram(@TempDir final Path tmp)
-        throws IOException {
-        final XML prev = OptCachedTest.program(ZonedDateTime.now(), "first program");
-        OptCachedTest.save(tmp, prev);
-        final XML current = OptCachedTest.program(ZonedDateTime.now(), "second program");
+    void returnsFromCacheCorrectProgram(@TempDir final Path cache, @TempDir final Path dir)
+        throws Exception {
+        final XML first = OptCachedTest.program("first program");
```

Comment:
@Yanich96 Can you inline this variable, please?
https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html